src/main/resources/edu/cornell/kfs/fp/document/validation/configuration/CU-DisbursementVoucherValidation.xml

```diff
@@ -98,4 +98,18 @@
 	</property>
 	</bean>
 	
+	<bean id="RecurringDisbursementVoucher-saveDocumentValidation" parent="DisbursementVoucher-saveDocumentValidation-parentBean">

```

Comment:
Should the foreign vendor check be happening at the save-document phase or the route-document phase? (Or both?)

Also, should the parent be the bean without the "-parentBean" suffix? Or is that needed for excluding any custom validation behavior on regular DVs that shouldn't be carried over to recurring DVs?

Reply:
it was originally implemented on the route phase, the user story I am working on would like this to happen on the save .... I added code so the validation isn't attempted if the vendor hasn't been selected yet. .... I will also try to not use the non-parent bean

Reply:
Ok, thanks for clarifying. I'm fine with this happening on the save accordingly.